quobyte_driver.go

```diff
@@ -38,11 +38,25 @@ func newQuobyteDriver(apiURL string, username string, password string, quobyteMo
 	return driver
 }
 
+func (driver quobyteDriver) StripVolumeName(requestName string) (strippedVolumeName string, subdirPath string) {
+	nameElems := strings.SplitN(requestName, "/", 2)
+	if len(nameElems) == 2 {
+		return nameElems[0], nameElems[1]
+	}
+	return nameElems[0], ""
+}
+
 func (driver quobyteDriver) Create(request volume.Request) volume.Response {
-	log.Printf("Creating volume %s\n", request.Name)
 	driver.m.Lock()
 	defer driver.m.Unlock()
 
+	strippedName, subDirs := driver.StripVolumeName(request.Name)
```

Comment:
I would prefer `volumeName` instead of `strippedName`

Reply:
done